src/xrpld/rpc/detail/RPCLedgerHelpers.cpp

```diff
@@ -269,18 +307,18 @@ getLedger(T& ledger, LedgerShortcut shortcut, Context& context)
     return Status::OK;
 }
 
-// Explicit instantiation of above three functions
+// Explicit instantiaion of above three functions
```

Comment:
```suggestion
// Explicit instantiation of above three functions
```

The original comment was fine.

Reply:
[`5af635f` (#5684)](https://github.com/XRPLF/rippled/pull/5684/commits/5af635feea5c9d18fffe9c71c5a835e52ffc8c9b)